modules/user_manual/pages/files/media_viewer_app.adoc

```diff
@@ -0,0 +1,59 @@
+= The Media Viewer App
+:browser-image-support-url: https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support
+:webm-url: https://www.webmproject.org/
+:ogg-url: https://xiph.org/vorbis/
+:mp4-url: https://en.wikipedia.org/wiki/MPEG-4_Part_14
+
+image:media-viewer-app/app-overview.gif[]
+
+== Introduction
+
+The Media Viewer app is a lightweight viewer for pictures and videos which integrates in the files app, and released under the GPLv2. 
+It replaces the gallery and files_videoplayer apps, which have now been deprecated, and supports the same basic feature set as the deprecated apps.
+
+IMPORTANT: Please remove the gallery and files_videoplayer apps before installing it.
+
+It supports the following functionality:
+
+* *Image*: _preview_, _zoom_, _rotation_, and _download_
+* *Video*: _playback_ and _download_
+
+NOTE: The app will support paginating through all media files in the current directory, even if only one media file was chosen to be previewed.
+
+== Features
```

Comment:
The part below seems to be copied from the gallery info.xml description. Some parts have changed. Please use the description from https://marketplace.owncloud.com/apps/files_mediaviewer here.